sky/serve/examples/vllm.yaml

```diff
@@ -1,21 +1,13 @@
 resources:
   accelerators: A100:1
-  cloud: gcp
-  # region: us-central1
-  # use_spot: True
-  image_id: projects/skypilot-375900/global/images/fastchat-serve-v0
-  
-num_nodes: 1
-
-file_mounts:
-  ~/chatlogs:
-    name: skypilot-chatbot-logs
-    store: gcs
-    mode: MOUNT
 
 service:
   port: 8081
-  readiness_probe: /health/v1/models
+  readiness_probe:
+    path: /v1/models
+    # vllm takes 5-10 minutes to install
+    initial_delay_seconds: 900
```

Comment:
Let's set a higher delay in case downloading model takes time. how about our default 20mins